RELEASE_INFO

```diff
@@ -1,6 +1,6 @@
 WIREDTIGER_VERSION_MAJOR=3
-WIREDTIGER_VERSION_MINOR=2
-WIREDTIGER_VERSION_PATCH=2
+WIREDTIGER_VERSION_MINOR=3
```

Comment:
My vote is for moving this major version out of a range where it can be confused with MongoDB. Could we rebase to 10.0.0? @lukech do you have an opinion and/or preference?

Reply:
Skipping the major version number is an unusual practice and needs some explanation for the community. The major architectural change of Durable history and the intention of avoiding confusion with MongoDB versions could be good reasons. That is to say, I'm not against this idea. 

And in my understanding, there're no mappings maintained between WiredTiger versions and log format versions in code, and we can defer the decision of WiredTiger version number till when we cut the next WiredTiger standalone release.

Reply:
@lukech, unfortunately there is a mapping maintained in `src/include/log.h`.

Reply:
@agorrod @lukech we will need to come to a decision on this before I can merge this change. There's logic in the code and python tests that depends on the WiredTiger version numbers, and I'll need to change the code logic if we bump the major version.

Reply:
Well, I actually don't have a strong opinion on the major version choice - there's much less attention to the WiredTiger standalone releases from the community compared to MongoDB releases. If bumping the major version to 10 could reduce our own mental overhead in the future let's just go for it. The code complexity could be another consideration (though it feels to me there won't be a major difference whether bumping major version or not).

Reply:
@keithbostic here's the conversation.

Reply:
@agorrod, @lukech, this change means we can no longer run the `mongodb-4.2` branch `wt` utility (or, well, any utility compiled with the `mongodb-4.2` branch), on objects created with the `develop` or `mongodb-4.4` branches. That means no meaningful upgrade/downgrade testing outside of MongoDB, specifically it breaks the Evergreen compat test.